pom.xml

```diff
@@ -216,6 +217,12 @@
         <artifactId>pentaho-cache-manager-api</artifactId>
```

Comment:
`pentaho-cache-manager-api` and `guava-cache-provider` were pulling in old mockito versions that got added to the classpath. We should check on these.

After we're done with all these upgrades, we should be able to remove these exclusions.